src/components/VirtualThought.tsx

```diff
@@ -149,11 +149,11 @@ const VirtualThought = ({
     onResize?.({
       height: heightNew,
       width: widthNew,
-      id: thought.id,
+      id: thought?.id,
```

Comment:
@raineorshine - It would be worth considering enabling `noUncheckedIndexedAccess` for TS. We currently look up the thought with `state.thoughts.thoughtIndex[id]`, and TS assumes that the index exists. With the flag enabled, the type of `thought` would be corrected from `Thought` to `Thought | undefined`.

Reply:
Yes, that's a good idea. We will have to consider the impact of that, as it could touch a lot of files. Would definitely help with `state.thoughts.thoughtIndex[id]`.